### PR TITLE
various typo corrections in test/e2e package

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/windows/gmsa_full.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ limitations under the License.
 //    the webhook's pod.
 //  * in order to run "can read and write file to remote folder" test case, a folder (e.g. "write_test") need to be created
 //    in that AD domain and it should be shared with that GMSA account.
-// All these assumptions are fulfilled by an AKS extension when setting up the AKS
```
Comment (dims) [inline test/e2e/windows/gmsa_full.go:37]: LOL. i think it is AKS :)

please go through once line by line.

Comment (humblec) [reply test/e2e/windows/gmsa_full.go]: actually I did @dims , missed this though

Comment (humblec) [reply test/e2e/windows/gmsa_full.go]: I have nullified the change @dims . ptal

**Comment by humblec** [comment]
@dims I had to `update-conformance` and refresh which took the lgtm away..

**Pull request merged**